code/game/objects/items/devices/radio/radio.dm

```diff
@@ -464,7 +464,11 @@ GLOBAL_LIST_INIT(default_medbay_channels, list(
 			// Simulate two seconds of lag
 			addtimer(CALLBACK(src, .proc/broadcast_callback, tcm), 2 SECONDS)
 		else
-			// Nukeops + Deathsquad headsets are instant and should work the same, whether there is comms or not
+			// Nukeops + Deathsquad headsets are instant and should work the same, whether there is comms or not, on all z levels
+			for(var/zlevel in 1 to length(GLOB.space_manager.z_list))
+				if(zlevel in tcm.zlevels)
+					continue
+				LAZYADD(tcm.zlevels, zlevel)
```

Comment:
Any reason youre using `LAZYADD` when it isnt a lazy list?

Reply:
because it freaking inserts it as an assoc value otherwise and I have no bloody clue why this makes it work.

Reply:
Have you considered trying
```dm
for(var/z in 1 to world.maxz)
    tcm.zlevels |= z
```

Reply:
Time to test!